evalml/tests/dependency_update_check/latest_dependency_versions.txt

```diff
@@ -14,7 +14,7 @@ ipywidgets==8.0.4
 kaleido==0.2.1
 lightgbm==3.3.5
 lime==0.2.0.1
-matplotlib==3.7.1
+matplotlib==3.7.2
```

Comment:
matplotlib just got updated to 3.7.2 so I changed that here but wasn't sure if I should be merging that to the main branch as well